res/enkf/enkf_main.py

```diff
@@ -297,7 +297,7 @@ def _init_res_config(self, config):
             config.convertToCReference(self)
             return config
 
-        raise TypeError("Expected ResConfig, received: %r" % config)
+        raise TypeError(f"Expected ResConfig, received: {config}")
```

Comment:
This one used the `__repr__` before but now `__str__` will be used, are these equivalent here?

Reply:
Nope, should be changed to `{config.__repr__()}` in order to be equivalent.

Reply:
Or `repr(config)`, but yes :slightly_smiling_face: